### DISCUSSION TITLE
Sorobanathon is for artists too!

### DISCUSSION THREAD

**Discussion by stellarjohn** [stellar/sorobanathon#66] [Announcements]
https://litemint.com/items/GBZLS2ALH3ZNAMZDRTU5FS53AA3OOQXGWZRVCD3ZIMRCFJHVQMYOR5UG/WIZARDIS

**Comment by achallagundla** [comment]
Check out this blog for more information 
https://medium.com/stellar-community/sorobanathon-is-for-artists-too-8b2da83c3997
